pkg/rtc/mediatrack.go

```diff
@@ -538,16 +332,20 @@ func (t *MediaTrack) ToProto() *livekit.TrackInfo {
 	return info
 }
 
+func (t *MediaTrack) GetAudioLevel() (level uint8, active bool) {
+	if t.audioLevel == nil {
```

Comment:
I think we can get away without locking here since only a single goroutine from Room was using it. perhaps we should add a lock now that it's an exported method.

Reply:
okay, will add one